docs/development.md

```diff
@@ -464,8 +464,16 @@ If you are firing a conversion event for a specific test, be sure that the `part
 |   `Prod`  |        `true`       |   `Dev`  | `Dev`    |       `Prod`      | `UAT`   |
 |   `Prod`  | `false` (real user) |  `Prod`  | `Prod`   |       `Prod`      | `Prod`  |
 
-## 10 CSS guidelines
+## 10 CSS
```

Comment:
By removing "guidelines", the link from the top of this file breaks and the link form the project's Readme file breaks too. Please update those.

Reply:
ah sorry!

Reply:
sorted

Reply:
... and the readme too .... sorry!